models/course_licensing/license_inventory.sql

```diff
@@ -0,0 +1,93 @@
+{{
```

Comment:
I don't really have much insight on the query aside from leaving some comments related to the operations performed on each SELECT, so it would be easy to understand what it's doing. @anfbermudezme